content/docs/apps/managing-teammates.md

```diff
@@ -15,10 +19,24 @@ Then, confirm with them that they have received and accepted the invite. Now the
 
 If you invited them with an agency email address and they're part of an agency that has automatic sandbox spaces, they'll also automatically get a sandbox space in their agency's cloud.gov org.
 
+### Option 2: cloud.gov Dashboard
+
+If you are an organization manager, you can invite teammates to cloud.gov directly from the Dashboard. All you need to invite someone is the email address they will use to login to cloud.gov.
```

Comment:
Can only org managers invite teammates from the dashboard? What about space managers?